epub33/rs/index.html

```diff
@@ -480,20 +480,20 @@ <h4>Metadata</h4>
 						<!-- <p>Reading System may use support linked resources, even if they recognize the
 							relationship defined in the <code>rel</code> attribute.
 						</p> -->
-						<p> The language identified in an <code>hreflang</code> is purely advisory. Upon fetching the
```

Comment:
Glad to see someone else shares my obsession with obliterating these damned spaces... 😉